action-templates/actions/action-maven-release/action.yml

```diff
@@ -80,12 +80,16 @@ runs:
         MVN_ARTIFACT_ID=$(mvn -f ./${{inputs.app-path}}/pom.xml org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.artifactId -q -DforceStdout)
         echo $MVN_ARTIFACT_ID
         echo "MVN_ARTIFACT_ID=$MVN_ARTIFACT_ID" >> $GITHUB_OUTPUT
+        # prepare the maven arguments
+        MAVEN_ARGS="-DpushChanges=${{ inputs.use-pr != 'true' }} -DremoteTagging=false -DlocalCheckout=true"
+        # set maven deploy skip if skipDeployment is true
+        if [[ "${{ inputs.skipDeployment }}" == "true" ]]; then
+          MAVEN_ARGS="$MAVEN_ARGS -Darguments=\"-Dmaven.deploy.skip=true\""
```

Comment:
> MAVEN_ARGS="$MAVEN_ARGS -Darguments=\"-Dmaven.deploy.skip=true\""

also `\"` finde ich nicht so schön, kann man das irgendwie anders machen? z. B. mit `'`

Reply:
Leider nicht, weil shell nur in `""` Evn-Variablen ersetzt und in `''` nicht

Reply:
aber `-Darguments=\"-Dmaven.deploy.skip=true\""` hat doch keine Variablen

Reply:
Geht um das `$MAVEN_ARGS` am Anfang